src/Layout/redist/targets/GenerateMSIs.targets

```diff
@@ -18,6 +18,12 @@
     <GenerateSdkBundleDependsOn Condition="'$(GenerateSdkBundleOnly)' != 'true'">$(GenerateSdkBundleDependsOn);GenerateSdkMsi;GenerateTemplatesMsis</GenerateSdkBundleDependsOn>
   </PropertyGroup>
 
+  <!-- In the VMR build we explicitly build workloads in build pass 2.
+       In the SDK official build we build workloads as part of win-x64 -->
+  <ItemGroup Condition="'$(DotNetBuild)' != 'true' and '$(BuildWorkloads)' == 'true'">
+    <ProjectReference Include="$(RepoRoot)src\Workloads\VSInsertion\workloads.csproj" ReferenceOutputAssembly="false" SkipGetTargetFrameworkProperties="true" Private="false" />
+  </ItemGroup>
```

Comment:
Please move this into the project file `src/Layout/redist/redist.csproj`. Defining it here makes it hard to spot the project dependency.